frontend/app/application.py

```diff
@@ -258,6 +258,7 @@ def project_index_template():
                        "yearEnd": get_year_from_date_string(project["dateEnd"])})
 
     return flask.render_template('project/project_index.html',
+                                 data_json=flask.Markup(json.dumps(titles)),
```

Comment:
Instead of `titles`, the variable name `template_data` includes more of its content

Reply:
all except first the arguments of render_template are template data, so I renamed `titles` to `projects`.

Reply:
good with me